frontend/app/views/info/supporterAustralia.scala.html

```diff
@@ -7,13 +7,20 @@
 @import com.gu.memsub.subsv2.Catalog._
 @import com.gu.memsub.subsv2.MonthYearPlans
 @import views.support.MembershipCompat._
+@import com.gu.salesforce.Tier.Supporter
+@import tracking.AppnexusPixel
 
 @import com.gu.memsub.subsv2.CatalogPlan
 @(heroImage: model.OrientatedImages, supporterPlans: MonthYearPlans[CatalogPlan.Supporter],
   pageInfo: PageInfo,
   pageImages: Seq[com.gu.memsub.images.ResponsiveImageGroup])(implicit token: play.filters.csrf.CSRF.Token, countryGroup: CountryGroup)
 
 @main("Supporters", pageInfo=pageInfo, countryGroup=Some(countryGroup)) {
+
+    <!-- Segment Pixel - Jan17-Membership Landing Page - DO NOT MODIFY -->
+        <img src="https://secure.adnxs.com/px?add=@{AppnexusPixel.landingPageids(Supporter())}&t=2" width="1" height="1" />
```

Comment:
Thank you for spotting that @pvighi, I will create a new card to refactor that after Christmas.